page-objects/src/components/bottomBar/WebviewView.ts

```diff
@@ -0,0 +1,21 @@
+import { WebElement, until } from "selenium-webdriver";
+import { AbstractElement } from "../AbstractElement";
+import WebviewMixin from "../WebviewMixin";
+
+/**
+ * Page object representing a user-contributed panel implemented using a Webview.
+ */
+class WebviewViewBase extends AbstractElement {
+
+    constructor() {
+        super(WebviewViewBase.locators.Workbench.constructor);
+    }
+
+    getViewToSwitchTo(handle: string): Promise<WebElement | undefined> {
```

Comment:
I believe this logic needs to be improved to retrieve real webelement which we want to switch to.. currently it is not working properly.. can be tested when you execute `webviewView-test` as single test